api/src/messaging/consumers.py

```diff
@@ -81,6 +82,35 @@ async def receive_message(self, text_data_json: dict) -> dict:
             "last_name": self.member["last_name"],
             "preferred_name": self.member["preferred_name"],
         }
+        content_type, content_object, object_id = (
+            text_data_json["content_type"],
+            None,
+            text_data_json["object_id"],
+        )
```

Comment:
Interesting syntax. Isn't this just 3 assignment statements? You used more lines for this.